### ISSUE TITLE
Menu cascade accessibility

### ISSUE THREAD

**Issue by wintrymix** [dita-ot/dita-ot#2446]
Common practice is to use a right angle bracket as part of a menu cascade: `File > New`.  In this case, the angle bracket is considered ASCII art, and per Web Content Accessibility Guidelines 2.0, should be avoided.

As a workaround for HTML, it is acceptable to wrap an abbreviate tag around the the angle bracket so screen readers will detect and read the content of the abbreviation tag: 

``` html
Choose File <abbr title="and then"> > </abbr> New

```

This issue is to request that we add processing to the HTML processing in the DITA OT to detect right angle brackets in menu cascades and wrap in abbreviation tags when publishing HTML.

**Comment by jelovirt** [comment]
Created #2516 to implement this.

**Comment by robander** [comment]
#2516 will be in release 2.5.

At the moment DITA-OT only ships with English text, but the variable is part of other strings, so when a DITA-OT user gets custom translations it will be part of the set.
